src/contexts/federated.ts

```diff
@@ -18,6 +25,10 @@ export type FederatedProps = Pick<DeleteInstanceProps, 'kafka'> & {
   showSchemas?: JSX.Element | undefined;
   redirectAfterDeleteInstance?: () => void;
   onDeleteInstance?: () => void;
+  kafka: KafkaRequest | undefined;
```

Comment:
I would specify what objects from kafka we need and avoid pass undefined.

Reply:
Actually I do not see any use case where we actually pass anything else than undefined. We might not need that field?